anyway/infographics_utils.py

```diff
@@ -234,6 +236,21 @@ def get_accident_count_by_severity(location_info, start_time, end_time):
         items["total_accidents_count"] = total_accidents_count
         return items
 
+    @staticmethod
+    def localize_items(request_params: RequestParams, items: Dict) -> Dict:
+        if request_params.lang != "en":
```

Comment:
Is it for performance?
we can do this check in `localize_after_cache()` once for all widgets.